<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>正常用户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>异常用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startCalcBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>580</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>重新生成</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>580</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>阈值：</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.200000000000000</double>
    </property>
    <property name="maximum">
     <double>0.900000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.300000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="calcProgressBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="normalTableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>581</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="abnormalTableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>581</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
